crud/src/main/java/com/redhat/lightblue/crud/CRUDFindRequest.java

```diff
@@ -187,9 +187,12 @@ public void fromJson(ObjectNode node) {
                 x=node.get("maxResults");
                 if(x!=null) {
                     long l=x.asLong();
-                    if(l>=0) {
+                    if (l > 0) {
                         to=(from==null?0:from)+l-1;
```

Comment:
@bserdar I am wondering if subtracting 1 is really what we want here? Setting maxResults to 0 calculates 'to' = -1. And maxResults of 1 calculates 'to' = 0.